validphys2/src/validphys/fitveto.py

```diff
@@ -75,9 +75,9 @@ def determine_vetoes(
         )
 
     # Positivity veto
-    posmask = np.array([replica.is_positive for replica in fitinfos], dtype=bool)
-    vetoes = {"Positivity": posmask}
-    total_mask = posmask.copy()
```

Comment:
Does this mean that no positivity veto is applied anymore? In such case would probably also remove the comment at line 77

Reply:
Ups, forgot the comment.

The veto hasn't changed, but since we decided to be strict on the positivity, the fit cannot converge if the positivity is not ok. And equally so, if the fit doesn't converge the positivity is not triggered. So calling it "positivity veto" was a misnomer.